src/library/Server/Manager/Directadmin.php

```diff
@@ -52,135 +74,192 @@ public static function getForm()
         );
     }
 
-    public function _getPort()
+    /**
+     * Returns the port number for the DirectAdmin server.
+     * If the port is not set in the configuration, it defaults to '2222'.
+     *
+     * @return float|int|string The port number.
+     */
+    public function _getPort(): float|int|string
```

Comment:
Why a float? A port can't be a float

Reply:
Because `is_numeric` can return a float if a float is passed by the user. I've replaced the function with `is_int` instead :)

Reply:
Ah, although, `is_int` will fail `'2222'`. Maybe sticking with `is_numeric` makes sense for now

Reply:
Well I was going to be happy with this `is_numeric` explanation, but I won't complain with a more robust implementation.
Thanks!